bin/postbuild.js

```diff
@@ -39,6 +41,9 @@ if (pathPrefix !== "") {
     exec(`mkdir "${dir}public"`, handleExec(() => {
       exec(`mv "${dir}${folderName}" "${dir}public/"`, handleExec(() => {
         console.log("Completed postbuild")
+        if (process.env.WORKERS_ENV && process.env.WORKERS_ENV === "development") {
+          fs.writeFile(`${process.env.PWD}/public/robots.txt`, robots, err => err && console.error(err))
```

Comment:
Maybe we should `console.log("Building robots.txt for development environment")` here.